gcivil-nyu-org/fall2019-cs-gy-6063-team-debug_entities
# PR for #59: Events Page More details

### Description

I added SeatGeek links to home, events, matches, and user profile pages.

In my opinion, the API doesn't give enough extra data to warrant a details page for each event, so I think linking to SeatGeek is a good solution.

### Testing Directions

```
python manage.py makemigrations
python manage.py migrate
python manage.py pull_seatgeek_data
python manage.py make_users
```

1. Make some matches
1. Go to the home, events, matches, and user profile pages and see that SeatGeek links have been added
![seatgeek links](https://user-images.githubusercontent.com/13837978/69928382-76893500-1489-11ea-8b32-7aa78bfb3a90.gif)

### Checklist
